packages/components/src/range-control/index.js

```diff
@@ -61,6 +61,7 @@ const BaseRangeControl = forwardRef(
 			onFocus = noop,
 			onMouseMove = noop,
 			onMouseLeave = noop,
+			resetFallbackValue,
```

Comment:
What's the prop needed for? Could we include documentation?

Reply:
> What's the prop needed for? Could we include documentation?

Still wondering about this one.

Reply:
Ah! Thanks for following up. It's the value to be used if the "Reset" button is clicked.

This was a prop that existed previously, but I guess didn't have docs. I'll add some.